authentication/utils.py

```diff
@@ -71,3 +71,22 @@ def is_user_email_blocked(email):
     """Returns the user's email blocked status"""
     hash_object = hashlib.md5(email.lower().encode("utf-8"))
     return BlockList.objects.filter(hashed_email=hash_object.hexdigest()).exists()
+
+
+def block_user_email(email):
+    """Blocked the user's email if provided"""
+    msg = None
+    if email:
```

Comment:
nit: we can also add email validation here.

Reply:
not necessary because we are fetching users from database in `retire_users` and `block_users` command. 
reference to `fetch_users(kwargs["users"])` method.